const-oid/oiddbgen/Cargo.lock

```diff
@@ -36,18 +36,18 @@ dependencies = [
 
 [[package]]
 name = "proc-macro2"
-version = "1.0.36"
```

Comment:
Could you revert changes on [const-oid/oiddbgen/Cargo.lock](https://github.com/RustCrypto/formats/pull/1051/files#diff-33d26b33538c98cd911ffec8d3685569bc7c07c8fa9cb58171b58c08f89ed50c) as well?